lib/operations/__tests__/linetv-list-modules-operation.spec.js

```diff
@@ -16,7 +17,26 @@ describe('linetv list:modules', () => {
     expect(LINETvListModulesOperation.usage).toEqual([
       {
         header: 'List curation module types'.help,
-        content: `linetv list:modules`.code
+        content:
+          `To display culation moodule types in table` +
```

Comment:
Typo here and several more..

Search for all ~~`culation`~~ change to `curation`
and all ~~`moodule`~~ change to `module`